test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -1188,4 +1244,33 @@ describe('v2/ion/uiSchemaTransformer', function () {
       });
     });
   });
+
+  it('converts seclect channel response for Okta verify', (done) => {
+    MockUtil.mockIntrospect(done, XHRAuthenticatorEnrollOktaVerifyQr, idxResp => {
+      const result = _.compose(uiSchemaTransformer, responseTransformer.bind(null, testContext.settings))(idxResp);
```

Comment:
Maybe a silly question, but where is the logic for this test? Is it part of this PR?

Reply:
ion-string-handler change so radio button is shown for channel